tests/framework/base/ComponentTest.php

```
@@ -10,405 +10,882 @@
+            $this->component->Text,
+            "Should return default value for 'Text' property.",
+        );
+        self::expectException(
+            UnknownPropertyException::class,
+        );
+        self::expectExceptionMessage(
+            'Getting unknown property: yiiunit\data\base\NewComponent::Caption',
+        );
+
         // @phpstan-ignore property.notFound
         $value2 = $this->component->Caption;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$value2'. (undefined)

(UnusedLocalVariable)